### PR TITLE
File added :MYSQL assignments 1,2,3

### PR DESCRIPTION
**Pull request by Priyankashelke2411**

### CODE REVIEW

**Review by shivnanda6272 — APPROVED** [review]

File: Assignments/MYSQL/Priyanka/PriyankaDB1_1.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Q4. Display all the fields from products table.
+SELECT * 
+FROM products;
+#Q5 Display the customer name,phone number,city,country of all the customers.
+SELECT customerName,phone,city,country 
+from customers;
+# Q6 Display the details of customers where the state is null.
+SELECT * 
+FROM customers
+WHERE state = null;
```
Comment (Snehal-113) [inline Assignments/MYSQL/Priyanka/PriyankaDB1_1.sql]: state IS NULL;

File: Assignments/MYSQL/Priyanka/PriyankaDB1_1.sql [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+FROM customers;
+#16. Display only those records whose creditlimit after inceasing by 2000 is between 70000 and 90000.
+SELECT  * ,creditLimit+2000
+FROM  customers
+WHERE (creditLimit+2000) >70000
+AND (creditLimit+2000) <85000;
+#USING BETWEEN statement 16. Display only those records whose creditlimit after inceasing by 2000 is between 70000 and 90000.
+SELECT  * ,creditLimit+2000
+FROM  customers
+WHERE creditLimit between 70000
```
Comment (Snehal-113) [inline Assignments/MYSQL/Priyanka/PriyankaDB1_1.sql]: (creditLimit+2000) BETWEEN 70000 and 80000

File: Assignments/MYSQL/Priyanka/PriyankaDB2.sql [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+# Q9. Display the first 6 employees if their firstname is sorted alphabetically.
+SELECT *
+FROM employees
+ORDER BY firstName
+LIMIT 0,6;
+# Q10. Display the customer details of the cutomers whose firstname starts with J and state is not null.
+SELECT *
+FROM customers
+WHERE (customerName LIKE 'J%' AND state != null);
```
Comment (Snehal-113) [inline Assignments/MYSQL/Priyanka/PriyankaDB2.sql]: state is not null

File: Assignments/MYSQL/Priyanka/PriyankaDB2.sql [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+descending order of creditlimit.*/
+SELECT *
+FROM customers
+WHERE (customerName LIKE 'J%' AND state != null)
+ORDER BY creditLimit DESC;
+/*Q12.Display the customer details of the first two cutomers whose firstname starts with J and state is not
+null in descending order of creditlimit.*/
+SELECT *
+FROM customers
+WHERE (customerName LIKE 'J%' AND state != null)
```
Comment (Snehal-113) [inline Assignments/MYSQL/Priyanka/PriyankaDB2.sql]: state is not null

File: Assignments/MYSQL/Priyanka/PriyankaDB2.sql [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+LIMIT 0,6;
+# Q10. Display the customer details of the cutomers whose firstname starts with J and state is not null.
+SELECT *
+FROM customers
+WHERE (customerName LIKE 'J%' AND state != null);
+/*Q11. Display the customer details of the cutomers whose firstname starts with J and state is not null in
+descending order of creditlimit.*/
+SELECT *
+FROM customers
+WHERE (customerName LIKE 'J%' AND state != null)
```
Comment (Snehal-113) [inline Assignments/MYSQL/Priyanka/PriyankaDB2.sql]: state is not null

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Review by yashubari — APPROVED** [review]

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**